.github/scripts/rust/setup.py

```diff
@@ -43,7 +43,7 @@
 ]
 
 # Build a coverage report for these crates
-COVERAGE_EXCLUDE_PATTERNS = ["apps/engine**"]
+COVERAGE_EXCLUDE_PATTERNS = []
```

Comment:
```suggestion
COVERAGE_EXCLUDE_PATTERNS = []
```

@TimDiekmann Do we need this line at all?

Reply:
This is still checked in the `setup.py` script so we could just add patterns here. I avoided any other changes than the removal.